src/components/setting/SetOfficeLocation.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useRef, useMemo, useState, useCallback } from 'react';
+import { MapContainer, Marker, Popup, TileLayer } from 'react-leaflet';
+
+const center = {
+  lat: 51.505,
+  lng: -0.09,
+};
+
+function DraggableMarker() {
+  const [draggable, setDraggable] = useState(false);
+  const [position, setPosition] = useState(center);
+  const markerRef = useRef(null);
+  const eventHandlers = useMemo(
+    () => ({
+      dragend() {
+        const marker = markerRef.current;
+        if (marker != null) {
+          setPosition(marker.getLatLng());
+        }
+      },
+    }),
+    [],
+  );
+  const toggleDraggable = useCallback(() => {
+    setDraggable(d => !d);
+  }, []);
+  return (
+    <div className='card'>
+      <Marker
+        draggable={draggable}
+        eventHandlers={eventHandlers}
+        position={position}
+        ref={markerRef}
+      >
+        <Popup minWidth={90}>
+          <span onClick={toggleDraggable} aria-hidden='true'>
+            {draggable ? 'Marker is draggable' : 'Click here to setting office location'}
```

Comment:
marker is dragable ganti ke move marker to office location

Reply:
done